src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1628503013.inc.php

```diff
@@ -0,0 +1,173 @@
+  ])
+  ->setWhereEquals([
+      'id' => '6b79ce5b-d7d9-7b4a-c774-104961d857c6',
+  ])
+;
+TCMSLogChange::update(__LINE__, $data);
+
+$data = TCMSLogChange::createMigrationQueryData('cms_tbl_display_list_fields', 'en')
+  ->setFields([
+      'title' => 'Titel',
+      'name' => '`cms_message_manager_type`.`name`',
+      'db_alias' => 'name',
+      'position' => '3',
+      'cms_tbl_conf_id' => TCMSLogChange::GetTableId('cms_message_manager_message_type'),
+      'cms_translation_field_name' => '',
+      'width' => '-1',
+      'align' => 'left',
+      'use_callback' => '0',
+      'show_in_list' => '1',
+      'show_in_sort' => '0',
+      'id' => '45a424da-644d-4226-f6e3-c439549d48c6',
+  ])
+;
+TCMSLogChange::insert(__LINE__, $data);
+
+$data = TCMSLogChange::createMigrationQueryData('cms_tbl_display_list_fields', 'de')
+  ->setFields([
+      'title' => 'Titel',
+  ])
+  ->setWhereEquals([
+      'id' => '45a424da-644d-4226-f6e3-c439549d48c6',
+  ])
+;
+TCMSLogChange::update(__LINE__, $data);
+
+$data = TCMSLogChange::createMigrationQueryData('cms_tbl_display_list_fields', 'en')
+  ->setFields([
+      'title' => 'ID',
+      'name' => '`cms_message_manager_type`.`id`',
+      'db_alias' => 'id',
+      'position' => '1',
+      'cms_tbl_conf_id' => TCMSLogChange::GetTableId('cms_message_manager_message_type'),
+      'cms_translation_field_name' => '',
+      'width' => '-1',
+      'align' => 'left',
+      'callback_fnc' => '',
+      'use_callback' => '0',
+      'show_in_list' => '1',
+      'show_in_sort' => '0',
+      'id' => '15c0df93-681e-5035-6519-6c3be86cb184',
+  ])
+;
+TCMSLogChange::insert(__LINE__, $data);
+
+$data = TCMSLogChange::createMigrationQueryData('cms_tbl_display_list_fields', 'de')
+  ->setFields([
+      'title' => 'ID',
+  ])
+  ->setWhereEquals([
+      'id' => '15c0df93-681e-5035-6519-6c3be86cb184',
+  ])
+;
+TCMSLogChange::update(__LINE__, $data);
+
+// set systemnames for common core message types
+$data = TCMSLogChange::createMigrationQueryData('cms_message_manager_message_type', 'en')
+    ->setFields([
+        'systemname' => 'unknown',
+    ])
+    ->setWhereEquals([
+        'id' => '1',
+    ])
+;
+TCMSLogChange::update(__LINE__, $data);
+
+$data = TCMSLogChange::createMigrationQueryData('cms_message_manager_message_type', 'en')
+    ->setFields([
+        'systemname' => 'notice',
+    ])
+    ->setWhereEquals([
+        'id' => '2',
+    ])
+;
+TCMSLogChange::update(__LINE__, $data);
+
+$data = TCMSLogChange::createMigrationQueryData('cms_message_manager_message_type', 'en')
+    ->setFields([
+        'systemname' => 'warning',
+    ])
+    ->setWhereEquals([
+        'id' => '3',
+    ])
+;
+TCMSLogChange::update(__LINE__, $data);
+
+$data = TCMSLogChange::createMigrationQueryData('cms_message_manager_message_type', 'en')
+    ->setFields([
+        'systemname' => 'error',
+    ])
+    ->setWhereEquals([
+        'id' => '4',
+    ])
+;
+TCMSLogChange::update(__LINE__, $data);
```

Comment:
There is also the "error conspicuous".

Reply:
I've added the error type with id 6 (ID 5 seems not to exist anywhere), but named it more common "error_striking"